treeherder/webapp/api/artifact.py

```diff
@@ -58,6 +59,9 @@ def create(self, request, project):
         job_guids = [x['job_guid'] for x in request.DATA]
         with JobsModel(project) as jobsModel, ArtifactsModel(project) as artifacts_model:
 
+            # create an accompanying ``Bug suggestions`` artifact for any eligible artifacts
```

Comment:
I think it would be more explicit if you did the check 
```python
artifact['name'] != 'text_log_summary'
```
here, so that you enter generate_bug_suggestions_artifacts only if you have to. And, just in case, you could assert this condition in the body of that function to catch eventual programming errors. @camd any thoughts?